packages/react-components/react-drawer/src/components/Drawer/useDrawer.ts

```diff
@@ -49,6 +61,7 @@ export const useDrawer_unstable = (props: DrawerProps, ref: React.Ref<HTMLElemen
     position = 'left',
     size = 'small',
     modal = true,
+    lightDismiss = true,
```

Comment:
Have you checked that the default behavior works? Looks like the Drawer is not closing by default anymore

Reply:
This is now fixed. I added also tests to ensure this.